### ISSUE TITLE
"hyperlink' doHook plugin events are not emitted 

### ISSUE THREAD

**Issue by boltex** [leo-editor/leo-editor#3758]
Not 100% sure about this , but I would have expected those to trigger when ctrl+clicking on UNLs 😄 

Maybe they were for some other thing. Feel free to close this issue if its the case.

**Comment by edreamleo** [comment]
@boltex A cff shows that there are no event triggers matching "hyperlink" anywhere!

From leoSettings.leo:
```
@bool underline-undefined-section-names = True
True: Underline undefined section names
Only functional if use_hyperlinks = 0

@bool use-hyperlinks = False
True:  underline "live" links.
False: (recommended): Underline undefined section names.
```
These settings appear to affect only Leo's jEdit colorizer.

**@url1 and @url2 events**

There are already `@url1` and `@url2` events. See `g.openUrl`.
But only `LeoTree.OnIconCtrlClick` and the `open-url` commands call this function!
  
**Summary**

- All documentation for "hyperlink" events (hooks) should be removed.
  I'll wait for your comments before doing so.
- I can add new events if you like, but there is no great need to do so.
- Handling clickable links in the log pane is incredibly complicated.
  I'm not going to change this logic as the result of this issue!

**Comment by boltex** [comment]
I guess it should just be removed from documentation as you've proposed :)

In the meantime, I've removed
        doHook('hypercclick1', { c: c, p: c.p, v: c.p });
and 
        doHook('hypercclick2', { c: c, p: c.p, v: c.p }); 

From leojs where I thought it might have been placed. (in openUrlOnClick)

**Comment by edreamleo** [comment]
@boltex Thanks for your comment. I'll just change the docs.

**Comment by edreamleo** [comment]
Closed via PR #3762.
